docs/dev-tools/zombienet/overview.md

```diff
@@ -235,66 +243,257 @@ Zombienet provides a CLI that allows interaction with the tool. The CLI can rece
-|              `timeout`               | Number  | Global timeout to use for spawning the whole network                                                      | -                                        |
-|              `provider`              | String  | Provider to use (e.g., kubernetes, podman)                                                                | kubernetes                               |
-|            `backchannel`             | Boolean | Deploy an instance of backchannel server. Only available on kubernetes                                    | `false`                                  |
-|       `polkadot_introspector`        | Boolean | Deploy an instance of polkadot-introspector. Only available on podman and kubernetes                      | `false`                                  |
-|            `jaeger_agent`            | String  | The Jaeger agent endpoint passed to the nodes. Only available on kubernetes                               | -                                        |
-|           `enable_tracing`           | Boolean | Enable the tracing system. Only available on kubernetes                                                   | `true`                                   |
-|        `tracing_collator_url`        | String  | The URL of the tracing collator used to query by the tracing assertion (Should be tempo query compatible) | -                                        |
-|   `tracing_collator_service_name`    | String  | Service name for tempo query frontend. Only available on kubernetes                                       | `tempo-tempo-distributed-query-frontend` |
-| `tracing_collator_service_namespace` | String  | Namespace where tempo is running. Only available on kubernetes                                            | `tempo`                                  |
-|   `tracing_collator_service_port`    | Number  | Port of the query instance of tempo. Only available on kubernetes                                         | `3100`                                   |
-|         `node_spawn_timeout`         | Number  | Timeout to spawn pod/process                                                                              | `per provider`                           |
-|              `local_ip`              | String  | IP used for exposing local services (rpc/metrics/monitors)                                                | `"127.0.0.1"`                            |
-|           `node_verifier`            | String  | Allow managing how to verify node readiness or disable (None)                                             | `Metric`                                 |
+Through the keyword `settings`, it's possible to define the general settings for the network. The available keys are:
+
+??? function "`bootnode` - add bootnode to network"
```

Comment:
For types,  I thought we discussed using the type formatting like on the XCM SDK. 

Instead, we should do this:

- `bootnode` ++"boolean"++ - add bootnode to network. Default is `true`
- `timeout` ++"number"++ - global timeout to use for spawning the whole network"
- `provider` ++"string"++ - provider to use. Default is `kubernetes`

etc.